src/Arcus.EventGrid.Tests.Integration/appsettings.json

```diff
@@ -1,12 +1,24 @@
 {
   "Arcus": {
-    "EventGrid": {
-      "TopicEndpoint": "#{Arcus_EventGrid_TopicEndpoint}#",
-      "EndpointKey": "#{Arcus_EventGrid_EndpointKey}#"
+    "EventGridEvent": {
+      "EventGrid": {
+        "TopicEndpoint": "#{EventGridEvent.EventGrid.TopicEndpoint}#",
+        "EndpointKey": "#{EventGridEvent.EventGrid.EndpointKey}#"
+      },
+      "ServiceBus": {
+        "TopicName": "#{EventGridEvent.ServiceBus.TopicName}#",
+        "ConnectionString": "#{EventGridEvent.ServiceBus.ConnectionString}#"
+      }
     },
-    "ServiceBus": {
-      "TopicName": "#{Arcus_ServiceBus_TopicName}#",
-      "ConnectionString": "#{Arcus_ServiceBus_ConnectionString}#"
+    "CloudEvent": {
+      "EventGrid": {
+        "TopicEndpoint": "#{CloudEvent.EventGrid.TopicEndpoint}#",
+        "EndpointKey": "#{CloudEvent.EventGrid.EndpointKey}#"
```

Comment:
@stijnmoreels  I've noticed these are still specific per schema type, can you move this above so that connection string is shared please? (topic will be specific per schema though)

Reply:
So specific for eventgrid topic but same service bus connection?
Will the tests then still work?

Reply:
It depends how you've configured it but they should share the same Service Bus namespace.

But maybe it's better that I'll pick this up with the Azure infra changes?

Reply:
Yes, I think so too 😞 . Thanks!!